subprojects/packagefiles/liburing/src/meson.build

```diff
@@ -11,6 +11,8 @@ liburing = library('uring',
                    c_args: '-DLIBURING_INTERNAL',
                    link_args: '-Wl,--version-script=' + meson.current_source_dir() + '/liburing.map',
                    link_depends: 'liburing.map',
+                   version: meson.project_version(),
+                   soversion: meson.project_version().split('.')[0],
```

Comment:
Per the soversion documentation:
> If this is not specified, the first part of `version` is used instead (see below). For example, if `version` is `3.6.0` and `soversion` is not defined, it is set to `3`.

Reply:
So you suggest dropping the `soversion` key word argument since we are reimplementing the default behavior?

Reply:
Exactly.